bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/items/DimmerItemTest.java

```diff
@@ -95,7 +95,7 @@ public void testAcceptedStates() {
     }
 
     private class DummyType extends HSBType {
-
+        private static final long serialVersionUID = -3327574976250687313L;
```

Comment:
what is the point of this class anyway? Testing the inheritance of of Java? Please remove.